e2e/tests/content-type-builder/ctb-edit-view.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { test, expect } from '@playwright/test';
+import { login } from '../../utils/login';
+import { navToHeader } from '../../utils/shared';
+import { resetDatabaseAndImportDataFromPath } from '../../scripts/dts-import';
+
+test.describe('Edit View CTB', () => {
+  test.beforeEach(async ({ page }) => {
+    await resetDatabaseAndImportDataFromPath('./e2e/data/with-admin.tar');
+    await page.goto('/admin');
+    await login({ page });
+  });
+
+  test('A user should be able to navigate to the Edit View of the content type builder and see the contentype fields', async ({
+    page,
+  }) => {
+    // Nabviagte to the content type builder
+    await navToHeader(page, ['Content-Type Builder'], 'Article');
+    const modal = page.getByRole('button', { name: 'Close' });
+    // Close the tour modal if it is visible
+    if (modal.isVisible()) {
+      modal.click();
+    }
+    // Navigate to the Shop single type and verify some essential buttons are visible
+    await page.getByRole('link', { name: 'Shop' }).click();
+    await expect(
+      page.getByRole('button', {
+        name: 'Add another field to this component',
+      })
+    ).toBeVisible();
+    await expect(page.getByRole('button', { name: 'Add a component' })).toBeVisible();
+    await expect(
```

Comment:
Glad you got the tests working! Could we add one that checks the button isn't visible on dzs?

I think you can use something like `expect(locator).toHaveCount(0)` on it, and to find it you can look for the button within the same row as "dynamic zone" so maybe something like `await expect(page.getByRole('row', { name: 'Dynamic Zone' }).getByRole('button', {name: 'Add a component'}).toHaveCount(0)`